BLSQ/iaso
# IA-2470 HOTFIX: org unit restore coordinates from logs

Restore functionaly from org unit logs wasn't working for coordinates(point)

Related JIRA tickets : IA-2470

## Self proofreading checklist


## Doc
In the code

## Changes

- parse location string into longitude and latitude
- only show the "keep changes only"  button in new version of log
- use clearer text on buttons

## How to test

Go to an org unit with a location (point).
If it has no history of changing the location, edit it from the "Map" tab
Go to the history tab:
- Restore only fields updated by this version button should only modify changed fields from that updates
- Restore all fields button will overwrite all fields of current org unit with the values from the logged version


## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/cde67e9b-0047-4dc8-893a-45ceb3e5fed9


Button text

![Screenshot 2024-06-26 at 12 52 54](https://github.com/BLSQ/iaso/assets/38907762/c3a59da1-549d-4502-b154-1d954d82a090)



## Notes

Since this PR updates some components that have been moved/deleted on main, a second one will be made to apply the changes correctly in the newer code

Conflicts with main will be handled in that second PR
